src/JK/WebsiteBundle/Controller/IndexController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace JK\WebsiteBundle\Controller;
+
+use JK\CommonBundle\Controller\AbstractController;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+/**
+ * @Route(service="jk.website.controller_index")
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @Route("/")
+     * @param string $name
+     * @return \Symfony\Component\HttpFoundation\Response
```

Comment:
Please don't use FQCN in annotations, just import it to current namespace and use directly class name